### PR TITLE
~nothing

### PR DESCRIPTION
**Pull request by hallucinomeny**
qol changes mostly

### CODE REVIEW

File: entropix/main.py [hunk 1]
```diff
@@ -267,6 +312,8 @@ def main():
   base_raw_tokens4 = tokenizer.encode(bp4, bos=True, eos=False, allowed_special='all')
 
 
+<<<<<<< HEAD
```
Comment (darrenangle) [inline entropix/main.py:315]: merge tags left in here

**Comment by pbarker** [comment]
does this check in the tokenizer model?

**Comment by xjdr-alt** [comment]
Going to cherry pick some of these vs merging this. First set of changes included in https://github.com/xjdr-alt/entropix/pull/11

**Comment by xjdr-alt** [comment]
closed in favor of https://github.com/xjdr-alt/entropix/pull/27

**Pull request closed without merging**